### ISSUE TITLE
[FlightDetails.Json]: Screen reader announcement is not descriptive enough for "SFO" and "JFK" and does not describe the topic or purpose of the element.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#94]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Turn on the Talkback.
2. Navigate to the Visual.
3. Navigate to the sample card.
4. Using swipe gesture, swipe left to right in the card.
5. Navigate to "SFO" and "JFK" text element.
6. Observe the issue.

**Actual Result:**
Screen reader announcement is not descriptive enough for "SFO" and "JFK" and does not describe the topic or purpose of the element. Also, there is no descriptive label is present under SFO and JFK.
Screen reader only announces "SFO" and "JFK" when focus moves to the said text block elements.

**Expected Result:**
Screen reader announcement should describe the purpose of "SFO" and "JFK" text elements.
For ex, Either Screen reader should announce like as 'Where from San Francisco international airport (SFO)' and 'Where to John F Kennedy international airport (JFK)' or below the SFO and JFK there should be descriptive label present so that when the focus lands on it then screen reader user can easily aware about the Airport short codes.  

**User Experience:**
The screen reader users will get impacted and might get confused when there is no heading or label info provided for text block element describing its purpose.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/570eeaf5-ba29-47d9-8a35-06a34625954d

![Screen reader announcement is not descriptive enough for SFO and JFK and does not describe the topic or purpose of the element](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/72e4ccf9-3cb7-4de7-9361-17d833e6e1f6)

### Card JSON

```JSON
NA
```
